tests/device_tests/test_msg_clearsession.py

```diff
@@ -17,77 +17,64 @@
 import pytest
 
 from trezorlib import messages as proto
+from trezorlib.btc import get_public_node
+from trezorlib.tools import parse_path
 
 
-@pytest.mark.skip_t2
-class TestMsgClearsession:
-    @pytest.mark.setup_client(pin=True, passphrase=True)
-    def test_clearsession(self, client):
-        with client:
-            client.set_expected_responses(
-                [
-                    proto.ButtonRequest(code=proto.ButtonRequestType.ProtectCall),
-                    proto.PinMatrixRequest(),
-                    proto.PassphraseRequest(),
-                    proto.Success(),
-                ]
-            )
-            res = client.ping(
-                "random data",
-                button_protection=True,
-                pin_protection=True,
-                passphrase_protection=True,
-            )
-            assert res == "random data"
+@pytest.mark.skip_ui
+@pytest.mark.setup_client(pin=True, passphrase=True)
+def test_clear_session(client):
+    if client.features.model == "1":
```

Comment:
@matejcik Maybe there is a nicer way?

Reply:
i don't think there is, let's keep it like this